tests/acceptance/features/webUISharingInternalGroups/shareWithGroups.feature

```diff
@@ -29,35 +31,39 @@ Feature: Sharing files and folders with internal groups
     Given user "user3" has logged in using the webUI
     When the user shares folder "simple-folder" with group "grp1" as "<set-role>" using the webUI
     And the user shares file "testimage.jpg" with group "grp1" as "<set-role>" using the webUI
+    And user "user1" accepts the share "simple-folder" offered by user "user3" using the sharing API
+    And user "user2" accepts the share "simple-folder" offered by user "user3" using the sharing API
+    And user "user1" accepts the share "testimage.jpg" offered by user "user3" using the sharing API
+    And user "user2" accepts the share "testimage.jpg" offered by user "user3" using the sharing API
     Then group "grp1" should be listed as "<expected-role>" in the collaborators list for folder "simple-folder" on the webUI
     And group "grp1" should be listed as "<expected-role>" in the collaborators list for file "testimage.jpg" on the webUI
     And user "user1" should have received a share with these details:
       | field       | value                |
       | uid_owner   | user3                |
       | share_with  | grp1                 |
-      | file_target | /simple-folder (2)   |
+      | file_target | /Shares/simple-folder  |
       | item_type   | folder               |
       | permissions | <permissions-folder> |
     And user "user2" should have received a share with these details:
       | field       | value              |
       | uid_owner   | user3              |
       | share_with  | grp1               |
-      | file_target | /testimage (2).jpg |
+      | file_target | /Shares/testimage.jpg |
       | item_type   | file               |
       | permissions | <permissions-file> |
-    And as "user1" these resources should be listed on the webUI
+    And as "user1" these resources should be listed in the folder "Shares" on the webUI
       | entry_name        |
-      | simple-folder (2) |
-      | testimage (2).jpg |
-    And these resources should be listed in the folder "simple-folder (2)" on the webUI
+      | simple-folder |
+      | testimage.jpg |
+    And these resources should be listed in the folder "/Shares%2Fsimple-folder" on the webUI
```

Comment:
This is a bit tricky - the first `/` is OK, and the 2nd one must be "escaped" as `%2F`
I guess the first one is stripped off anyway by the test code?

Reply:
Yes, I guess the test places the string directly at the end of browser's URL. So they need  to be escaped.